solutions/streamproc_azure_kafka/index.asciidoc

```diff
@@ -8,6 +8,10 @@ toc::[]
 :idprefix:
 :idseparator: -
 
+include::../../includes/streamproc_problem/index.asciidoc[]
```

Comment:
Please do not use includes as a first content in solutions, because the solution is then named by the header in the include. That can lead to multiple solutions with the same name, when new solutions use the include in the same way.

Either change the structure of the document and the include or add a first level header above
E.g.:
```
= Apache Kafka on Microsoft Azure

include::../../includes/streamproc_problem/index.asciidoc[]
...
```